Automattic/wp-calypso
# Remove difm upsell experiment code

#### Changes proposed in this Pull Request

* The DIFM upsell experiment was launched in https://github.com/Automattic/wp-calypso/pull/49301. The result of the test is to retain control, check pcbrnV-12W-p2 for more details. This PR removes the experiment code and retains control.

#### Testing instructions



* Go through the signup flow and purchase a plan, you should see a post purchase Quick Start upsell for Business and eCommerce plans, and a Business plan bump offer for the Premium plan. 
* Verify no JS errors.
* Cross-check with the PR that introduced this experiment https://github.com/Automattic/wp-calypso/pull/49301 to confirm that all experiment code has been removed
